__docs__/wonder-blocks-form/_overview_.mdx

```diff
@@ -49,6 +49,12 @@ user is done interacting with a field.
 button is for one field.
 - If there are errors after a form is submitted, programatically move the user's
 focus to the first field with an error.
+- Accessibility: When using a `<form>` element to wrap form fields, avoid using
+event handlers (such as key events) directly on the `<form>` element since it
+is non-interactive. To submit a form when the `Enter` key is pressed,
+include a `Button` with `type="submit"` inside the form and handle submission
+using the `onSubmit` prop on the `<form>` element. The "Enter to submit" functionality
+is handled by the browser.
```

Comment:
This is related to a common jsx-a11y error in `webapp`: [no-noninteractive-element-interactions](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-element-interactions.md). `<form>` elements (which are non-interactive) would have key event handlers to handle pressing "Enter" to submit a form. This guideline can help address that issue (I also mentioned it in the [Tips section of the a11y lint rules blog post](https://khanacademy.atlassian.net/wiki/x/Z4C64)!)